backend/src/main/java/com/happy/friendogly/playground/service/PlaygroundCommandService.java

```diff
@@ -1,22 +1,79 @@
+import com.happy.friendogly.playground.domain.PlaygroundMember;
+import com.happy.friendogly.playground.dto.request.SavePlaygroundRequest;
+import com.happy.friendogly.playground.dto.response.SavePlaygroundResponse;
 import com.happy.friendogly.playground.repository.PlaygroundMemberRepository;
 import com.happy.friendogly.playground.repository.PlaygroundRepository;
+import com.happy.friendogly.utils.GeoCalculator;
+import java.util.List;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @Transactional
 public class PlaygroundCommandService {
 
+    public static final int PLAYGROUND_EXCEPTION_DISTANCE_THRESHOLD = 300;
+
     private final PlaygroundRepository playgroundRepository;
     private final PlaygroundMemberRepository playgroundMemberRepository;
+    private final MemberRepository memberRepository;
 
     public PlaygroundCommandService(
             PlaygroundRepository playgroundRepository,
-            PlaygroundMemberRepository playgroundMemberRepository
-    ) {
+            PlaygroundMemberRepository playgroundMemberRepository,
+            MemberRepository memberRepository) {
         this.playgroundRepository = playgroundRepository;
         this.playgroundMemberRepository = playgroundMemberRepository;
+        this.memberRepository = memberRepository;
+    }
+
+    public SavePlaygroundResponse save(SavePlaygroundRequest request, Long memberId) {
+        Member member = memberRepository.getById(memberId);
+
+        validateExistParticipatingPlayground(member);
+        validateOverlapPlayground(request.latitude(), request.longitude());
+
+        Playground savedPlayground = playgroundRepository.save(
+                new Playground(new Location(request.latitude(), request.longitude()))
+        );
+        playgroundMemberRepository.save(new PlaygroundMember(savedPlayground, member));
+
+        return SavePlaygroundResponse.from(savedPlayground);
+    }
+
+    private void validateExistParticipatingPlayground(Member member) {
+        if (playgroundMemberRepository.existsByMemberId(member.getId())) {
+            throw new FriendoglyException("이미 참여한 놀이터가 존재합니다.");
+        }
+    }
+
+    private void validateOverlapPlayground(double latitude, double longitude) {
+        Location location = new Location(latitude, longitude);
+        double startLatitude = GeoCalculator.calculateLatitudeOffset(latitude,
+                -PLAYGROUND_EXCEPTION_DISTANCE_THRESHOLD);
+        double endLatitude = GeoCalculator.calculateLatitudeOffset(latitude,
+                PLAYGROUND_EXCEPTION_DISTANCE_THRESHOLD);
+        double startLongitude = GeoCalculator.calculateLongitudeOffset(latitude, longitude,
+                -PLAYGROUND_EXCEPTION_DISTANCE_THRESHOLD);
+        double endLongitude = GeoCalculator.calculateLongitudeOffset(latitude, longitude,
+                PLAYGROUND_EXCEPTION_DISTANCE_THRESHOLD);
```

Comment:
```suggestion
        Location location = new Location(latitude, longitude);
        double startLatitude = location.minusLatitude(PLAYGROUND_RADIUS_METER);
        double endLatitude = location.plusLatitude(PLAYGROUND_RADIUS_METER);
        double startLongitude = location.minusLongitude(PLAYGROUND_RADIUS_METER);
        double endLongitude = location.plusLongitude(PLAYGROUND_RADIUS_METER);
```

그럼 이렇게 쓸 수 있을듯! `Location` 정의해놓은 이상 최대한 활용하면 좋을 것 같아서요